app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+
+from app.errors import (
+    OutdatedVaccineError,
+    NotWearingMaskError,
+    NotVaccinatedError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        date_today = datetime.date.today()
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError("NotVaccinatedError")
+        if visitor["vaccine"]["expiration_date"] < date_today:
+            raise OutdatedVaccineError("OutdatedVaccineError")
+        if visitor.get("wearing_a_mask") is False:
```

Comment:
Okay, but for example visitor.get("wearing_a_mask") == "" it is means that user have mask???
Also btw if .get will return None
None is not False
```suggestion
        if not visitor.get("wearing_a_mask"):
```